### PR TITLE
🔧 z-index to tailwind

### LINKED ISSUE #9004: remove `z-index` on global.scss
Related issue: https://github.com/kodadot/nft-gallery/issues/8071

### PR DESCRIPTION
**Pull request by roiLeo**
- close #9004

### CODE REVIEW

File: components/collection/drop/GenerativePreview.vue [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <template>
   <div class="fixed-size border">
     <div
-      class="fixed-top-left border px-4 py-2 bg-background-color has-z-index-1 whitespace-nowrap">
+      class="fixed-top-left border px-4 py-2 bg-background-color z-10 whitespace-nowrap">
```
Comment (Jarsen136) [inline components/collection/drop/GenerativePreview.vue:4]: why not `z-[1]` as it's `has-z-index-1` before

**Review by Jarsen136 — APPROVED** [review]

File: components/collection/drop/GenerativePreview.vue (hunk 1 above)
Comment (roiLeo) [reply components/collection/drop/GenerativePreview.vue:4]: > why not `z-[1]` as it's `has-z-index-1` before

we are now using tailwind, `has-z-index-1` has now been removed for more consistency

Comment (Jarsen136) [reply components/collection/drop/GenerativePreview.vue:4]: yup, I mean why not use `z-[1]` instead of `z-10`

Comment (roiLeo) [reply components/collection/drop/GenerativePreview.vue:4]: `z-10` feels better thant `z-[1]`, it's the only one I've change and it worked well so far

Comment (roiLeo) [inline components/collection/drop/GenerativePreview.vue:4]:
```suggestion
      class="fixed-top-left border px-4 py-2 bg-background-color z-[1] whitespace-nowrap">
```

Comment (Jarsen136) [reply components/collection/drop/GenerativePreview.vue]: Okay

**Pull request merged**